tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "allowSyntheticDefaultImports": true,
     "strict": true,
     "forceConsistentCasingInFileNames": true,
-    "module": "esnext",
+    "module": "CommonJS",
```

Comment:
I wonder if it is worth maintaining two copies, one cjs and one umd, etc in case people do not want to use node/webpack? We can explore this in another diff.